cas/incident-report.md

```diff
@@ -37,6 +37,8 @@ Incident reports should be updated when:
 * Completion of a resolution step, or
 * Delays in completing a resolution step.
 
+If the bug is still open, then you should also provide updates at least every week giving your progress, until you confirm that the remediation steps have been completed, unless a root store representative has agreed to a different schedule by setting a “Next Update” date in the “Whiteboard” field of the bug or has announced they consider closing the bug and no further comments have been posted.
```

Comment:
Minor nits: 
- change "remediation steps" to "resolution steps" for consistency with the above statements
- change "root store representative" to "Root Store Operator" for consistency with the rest of ccadb.org